structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,249 @@
+
+The poll should be as detailed as possible in order to help everyone to make an informed decision. Discussions related to the poll should happen in the associated thread, to try to keep everything together.
+
+Each community is assigned one single vote. Individual communities are responsible for coming up with their own process for determining their vote, but it must follow these rules:
+
+* The process must be democratic and give a voice to any community member that is otherwise eligible to join the Program's Discord server under the Collaborator role, even if they're not currently present on the server
+* Each eligible member of the community must have equal voting power
+* Eligible members may only be excluded via a reasonable inactivity threshold
+* The process must be formally documented somewhere, but this documentation may be kept internal to the given community
+* Participating communities must allow the Program's Manager team to audit their process, which may be motivated by reports from the given community's members, or from other Program members, including the Managers themselves
+
+Once a community has decided how it's going to vote, it must designate a single eligible community member to represent that vote. This representative must report their community's vote in the associated thread, specifically mentioning the community's name and the option being voted for, and providing proof of the results --- such as a screenshot of an internal poll.
+
+Participating communities also have the option of abstaining from the vote. This must either be reported in the thread as described above or, in the case of a community that wishes to abstain from all votes, directly to a Program Manager in advance. Regardless of the method, documented consensus among eligible community members must be provided.
+
+All polls must include a closing date that is set at least ten days into the future. If needed, this date may be pushed back by up to a month, as long as an informal agreement between the majority of the Program's participating, non-abstaining communities has been reached, and documented within the thread associated with the poll.
+
+At any point before the poll has closed, any Collaborator can use their right to veto, if allowed by the process. This right is described below.
+
+Once the poll has been closed, the vote should be interpreted as described by the process the poll is a part of.
+
+**Note:** Individual Collaborators currently do not hold the right to vote.
+
+### The right to veto
+
+While a poll is open and accepting votes, any Collaborator or Manager can use their right to veto by sending a message to the associated thread, detailing why they'd like to veto the vote.
```

Comment:
Allowing individuals to veto seems like it would defeat the point of communities voting together. It effectively means any single person who disagrees with the rest of their community can stop the vote and move to voting on an individual basis.

Maybe we should require an entire community to agree to veto?